charts/shoot-core/charts/csi-packet/templates/node.yaml

```diff
@@ -0,0 +1,131 @@
+kind: DaemonSet
+apiVersion: {{ include "daemonsetversion" . }}
+metadata:
+  name: csi-node
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  selector:
+    matchLabels:
+      app: csi-packet-driver
+  template:
+    metadata:
```

Comment:
You might want to inject a checksum annotation for the secret so that this pod is automatically restarted if the secret changes: https://github.com/gardener/gardener/blob/master/charts/shoot-core/charts/csi-alicloud/templates/csi-diskplugin-ds.yaml#L19

Reply:
Done.